test/pkg/test/framework.go

```diff
@@ -86,7 +89,7 @@ func NewFramework(ctx context.Context) *Framework {
 	SetDefaultEventuallyTimeout(180 * time.Second)
 	SetDefaultEventuallyPollingInterval(10 * time.Second)
 	BeforeEach(func() { framework.ExpectToBeClean(ctx) })
-	AfterEach(func() { framework.ExpectToBeClean(ctx) })
+	AfterEach(func() { framework.CleanTestEnvironment(ctx); framework.ExpectToBeClean(ctx) })
```

Comment:
Please revert this change.  We purposely do it as `pause on failure` and keep all lattice resource(s), so that we can debug in case of failure!

Reply:
This was included totally by mistake, I will remove it.